packages/config/src/bin/dev-script.ts

```diff
@@ -15,12 +15,23 @@ async function setEnv(cwd: string) {
         type: 'list',
         name: 'clouddotEnv',
         message: 'Which platform environment you want to use?',
-        choices: ['stage', 'prod', 'dev'],
+        choices: ['stage', 'prod', 'dev', 'ephemeral'],
       },
     ])
-    .then((answers) => {
+    .then(async (answers) => {
       const { clouddotEnv } = answers;
-      process.env.CLOUDOT_ENV = clouddotEnv ? clouddotEnv : 'stage';
+
+      if (clouddotEnv === 'ephemeral') {
+        const answer = await inquirer.prompt([{
+          type: 'input',
+          name: 'clouddotEnv',
+          message: 'Please provide the URL of your ephemeral environment:',
+        }]);
+        process.env.EPHEMERAL_TARGET = answer.clouddotEnv
+        process.env.CLOUDOT_ENV = 'stage';
```

Comment:
@riccardo-forina are we intentionally setting "stage" here? Shouldn't this be ephemeral as well?

Reply:
Uhm i was being lazy. If we use ephemeral we end up with ephemeral.foo.redhat.com. I think we’ll have to update the hosts file (and the patch script) and I’m not 100% sure chrome will be happy about it

Reply:
I’ll try tomorrow tho

Reply:
🤦 I forgot to update the hosts file, that's why it was hanging...